doc/cli/introduction.md

```diff
@@ -9,21 +10,76 @@ Nevertheless, when it comes to initial deployment and debugging, it
+
+New users usually get the CLI as the default "shell" when logging in,
+but the default `admin` user logs in to `bash`.  To access the CLI,
+type:
+
+```
+admin@host-12-34-56:~$ cli
+
+See the 'help' command for an introduction to the system
+
+admin@host-12-34-56:/>
+```
+
+The prompt (beginning of the line) changes slightly.  Key commands
+available in any context are:
+
+```
+admin@host-12-34-56:/> help                   # Try: Tab or ?
+...
+admin@host-12-34-56:/> show                   # Try: Tab or ?
+admin@host-12-34-56:/>                        # Try: Tab or ?
+```
+
+> **Tip:** Even on an empty command line you can tap the Tab or ? keys.
+> See `help keybindings` for more tips!
+
+
 ## Key Concepts
 
 The two modes in the CLI are the admin-exec and the configure context.
 
-When logging in to the system, be it from console or SSH, you first have
-a stopover in a UNIX shell, usually Bash.  Type `cli` to enter the CLI
-and finally land in admin-exec.  Here you can inspect system status and
-do operations to debug networking issues, e.g. ping.  You can also enter
-configure context by typing: `configure`
+The top-level context after logging in and starting the CLI is the
+admin-exec or "main" context.  It is used for querying system status,
+managing configuration files/profiles and doing advanced debugging.
+
+Available commands can be seen by pressing `?` at the prompt:
 
-The system has several datastores (or files):
+```
+admin@host:/>
+  configure      Create new candidate-config based on running-config
+  copy           Copy file or configuration, e.g., copy running-config startup-config
+  dir            List available configuration files
+  exit           Exit from CLI (log out)
+  factory-reset  Restore the system to factory default state
+  follow         Monitor a log file, use Ctrl-C to abort
+  help           Help system (try also the '?' key)
+  logout         Alias to exit
+  netcalc        IP subnet calculator, with subnetting
+  password       Password tools
+  ping           Ping a network host or multicast group
+  poweroff       Poweroff system (system policy may yield reboot)
+  reboot         Reboot system
+  remove         Remove a configuration file
+  set            Set operations, e.g., current date/time
+  show           Show system status and configuration files
+  tcpdump        Capture network traffic
+  upgrade        Install a software update bundle from remote or local file
+```
+
+The system has three *main datastores* (or files): *factory*, *startup*,
+and *running* that can be managed and inspected using the `copy`,
+`show`, and `configure` commands.  The traditional names used in the CLI
+for these are listed below:
 
  - `factory-config` consists of a set of default configurations, some
    static and others generated per-device, e.g., a unique hostname and
-   number of ports/interfaces.   This file is generated at boot.
+   number of ports/interfaces.   This file is generated at boot and is
```

Comment:
Pretty soon, this won't be true for all systems. Do we either want to mention that, or remove the bit about it being generated altogether? (It does seem like a bit of an implementation detail :man_shrugging:)

Reply:
Good point, I'll simplify it, thanks!